### PR TITLE
add option to use random tokens for python benchmarking

### PR DESCRIPTION
**Pull request by thevishalagarwal**
add option to use random set of tokens for benchmark instead of generating a prompt with the help of the LLM. This option helps to reduce benchmark time.

### CODE REVIEW

File: benchmark/python/benchmark_e2e.py [hunk 1]
```diff
@@ -225,8 +227,13 @@ def run_benchmark(args, batch_size, prompt_length, generation_length, max_length
 
  
     # Generate prompt
-    prompt = [generate_prompt(model, tokenizer, prompt_length, args.use_graph_capture)] * batch_size
-    tokens = tokenizer.encode_batch(prompt)
+    if args.use_random_tokens:
+        # use random tokens instead of generating a prompt using the model and then tokenizing it
+        tokens = np.random.randint(100, size=(batch_size, prompt_length))
+        prompt = [tokenizer.decode(tokens[0])]*batch_size
```
Comment (baijumeswani) [inline benchmark/python/benchmark_e2e.py:233]:
```suggestion
        prompt = [tokenizer.decode(tokens[0])] * batch_size
```

File: benchmark/python/benchmark_e2e.py [hunk 2]
```diff
@@ -225,8 +227,13 @@ def run_benchmark(args, batch_size, prompt_length, generation_length, max_length
 
  
     # Generate prompt
-    prompt = [generate_prompt(model, tokenizer, prompt_length, args.use_graph_capture)] * batch_size
-    tokens = tokenizer.encode_batch(prompt)
+    if args.use_random_tokens:
```
Comment (baijumeswani) [inline benchmark/python/benchmark_e2e.py:230]:
```suggestion
    tokens, prompt = None, None
    if args.use_random_tokens:
```

**Comment by baijumeswani** [comment]
Minor comments. otherwise, the pull-request looks good.

**Comment by thevishalagarwal** [comment]
Thanks for the review

**Review by baijumeswani — APPROVED** [review]

**Pull request merged**